README.md

```diff
@@ -367,6 +367,11 @@ Remark42 supports importing comments from Disqus, WordPress, or native backup fo
 2. Move this file to your Remark42 host within `./var`
 3. Run import command - `docker exec -it remark42 import -p wordpress -f /srv/var/{wordpress-export-name}.xml -s {your site ID}`
 
+##### Initial import from Commento
```

Comment:
Should be duplicated to site/src/docs/backup/migration/index.md as well, as we're in progress of migrating the documentation to the [separate site](https://remark42.com/docs/).

Reply:
done